lib/kitchen/verifier/pester.rb

```diff
@@ -387,14 +520,22 @@ def restart_winrm_service
         CMD
                                      ))
       end
-
+      
       def download_test_files(state)
-        return if config[:downloads].nil?
+        if config[:downloads].nil?
+          info("Not downloading test result files from #{instance.to_str}.")
+          return
+        end
 
         info("Downloading test result files from #{instance.to_str}")
         instance.transport.connection(state) do |conn|
-          config[:downloads].to_h.each do |remotes, local|
-            debug("Downloading #{Array(remotes).join(", ")} to #{local}")
+          config[:downloads].each do |remotes, local|
+            
+            # if the destination ends by / or \ make sure it's a folder and it's created
+            # if local.match?('\\$|/$')
+            #   info("Local folder: #{local}")
+            #   FileUtils.mkdir_p(local)
+            # end
```

Comment:
Is this commented code needed?

I also think the debug logging here is worth keeping; is there a reason we'd want to remove it?

Reply:
The logging is done in the `resolve_downloads_paths!` now, error would give enough info IMO.

Reply:
Will remove the comments, sorry, was probably too excited that it finally was working :)

Reply:
fixed